components/cam/src/physics/cam/bfb_math.inc

```diff
@@ -6,8 +6,8 @@
 ! Make sure to place the following lines at the top of any modules
 ! that use these macros:
 !
-! use physics_common_iso_f, only: cxx_pow, cxx_sqrt, cxx_cbrt, cxx_gamma, cxx_log, &
-!                                 cxx_log10, cxx_exp
+! use physics_share_iso_f, only: cxx_pow, cxx_sqrt, cxx_cbrt, cxx_gamma, cxx_log, &
```

Comment:
Is it too invasive of a modification to add the change from `_iso_f` to `_f2c`, as we discussed somewhere (maybe confluence)? Imho, it's perfectly ok to leave that change for another PR, but since you were renaming things...

Reply:
I can do that.